python/HeaderService/hslib_salobj.py

```diff
@@ -461,10 +461,10 @@ def collect_from_HeaderService(self):
             self.metadata['RASTART'] = ra
             self.metadata['DECSTART'] = dec
         else:
-            self.log.info("No 'ELSTART' and 'AZSTART'")
+            self.log.info("No 'ELSTART' and 'AZSTART' to compute RA/DEC START")
```

Comment:
This implies that neither `ELSTART` nor `AZSTART` is in the header.  Please change the message to be specific about which key[s] are missing

Reply:
Fixed as requested.